### PR TITLE
Slack integration

### PR DESCRIPTION
**Pull request by Skarlso**

### CODE REVIEW

**Comment by Skarlso** [comment]
@todbot  Hi! 

Created something... :) Would you mind taking a look? It doesn't appear to be running though. I don't see any errors in the log ( I found a few after successfully running dev mode ).

It looks like the rtm service isn't actually running because bot is offline.

File: package.json [hunk 1]
```diff
@@ -132,5 +132,12 @@
     "webpack": "^4.31.0",
     "webpack-cli": "^3.3.2",
     "webpack-dev-server": "^3.4.1"
+  },
+  "dependencies": {
+    "@slack/events-api": "^2.3.2",
```
Comment (Skarlso) [inline package.json:137]: the events and web-api are not needed I'll remove them later on

**Comment by Skarlso** [comment]
@todbot This now works!! With a classic app it works! :))))) I'm going to write up a guide in the README on how to set it up. :) But it works! \o/

**Comment by applefreak** [comment]
Nice job man!

**Comment by Skarlso** [comment]
Thanks man!

**Comment by Skarlso** [comment]
@todbot Hey, would you like me to open this PR to the new main branch?

**Comment by Skarlso** [comment]
No worries. :)

**Pull request closed without merging**